src/client/pages/Debugger/components/QuoteFormSweden.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { ApartmentType } from 'data/graphql'
+import { InputField } from 'components/inputs'
+import { WithFormikProps } from './Offer'
+
+export const initialSeApartmentValues = {
+  firstName: 'Sven',
+  lastName: 'Svensson',
+  currentInsurer: '',
+  birthDate: '1995-09-29',
+  ssn: '199509291234',
+  startDate: '',
+  email: 'sven.svensson@hedvig.com',
+  swedishApartment: {
+    street: 'Storgatan 1',
+    zipCode: '12345',
+    livingSpace: 23,
+    householdSize: 1,
+    type: ApartmentType.Rent,
+  },
+}
+
+export const SwedishApartment: React.FC<WithFormikProps> = ({ formik }) => {
+  return (
+    <>
+      <InputField
+        label="Household size"
+        placeholder="1"
+        type="number"
+        {...formik.getFieldProps('swedishApartment.householdSize')}
+      />
+      <InputField
+        label="Living space"
+        placeholder="23"
+        type="number"
+        {...formik.getFieldProps('swedishApartment.livingSpace')}
+      />
+      <InputField
+        label="Street"
+        placeholder="Gulebøjsveien 1"
+        {...formik.getFieldProps('swedishApartment.street')}
+      />
+      <InputField
+        label="Zip code"
+        placeholder="12345"
+        {...formik.getFieldProps('swedishApartment.zipCode')}
+      />
+      <InputField
+        label="Type"
+        placeholder=""
```

Comment:
is this required?

Reply:
For Sweden it is required to pass the type in order to create a quote yes. Or maybe you mean we should actually add `required` to all input fields that are required?

Reply:
Or did you mean the `placeholder`? I don't remember, since those were already there when I started working one this I assumed it's something required by Formik. I'll look into it!

Reply:
OK so it's required in the `interface CoreInputFieldProps` in `/inputs/index.tsx`. Can I make that property optional maybe? Or will that mess up something else?

Reply:
I think we should make it optional, empty string just feels silly

Reply:
👍